src/inferenceql/query/cache.cljc

```diff
@@ -0,0 +1,20 @@
+(ns inferenceql.query.cache
+  "For caching expensive results."
+  (:require #?(:clj [clojure.core.memoize :as memo]
+               :cljs [memoizee :as memoizee])
```

Comment:
we might consider using strings for js/npm dependencies (eg. `["memoizee" as memoizee]`) so that it's always apparent when a dep is coming from javascript vs the cljs classpath. Symbols work, but don't cover all cases (eg. scoped packages like `"@foo/bar"` or `some-module/path`) and it can be handy to differentiate at a glance.

Reply:
Sure, I have no strong opinion on this. I originally left it as a symbol because I knew it would work here.

Reply:
Changed